src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
 public class GreetingController {
 
```

Comment:
3. Unit Testing:
It's recommended to write unit tests for these methods to ensure they behave as expected. Testing both the successful path (for the `greeting` method) and the error path (for the `throwError` method) will help ensure the reliability of the code.